core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -319,13 +319,24 @@ public Options fetch(ColumnType... colsToFetch) {
       return this;
     }
 
+    /**
+     * Set the range to the table range that stores the provided level's metadata. For example
+     * {@link DataLevel#USER} will set the range to the metadata table. {@link DataLevel#METADATA}
+     * will set the range to the root table.
+     */
```

Comment:
```suggestion
    /**
     * For a given data level, read all of its tablets metadata.  For DataLevel.USER this will read tablet metadata from the accumulo.metadata table.  For DataLevel.METADATA this will read tablet metadata from the accumulo.root table.  For DataLevel.ROOT this will read tablet metadata from Zookeeper.
     */
```